source/content/guides/composer-convert.md

```diff
@@ -92,18 +128,15 @@ A Composer-managed site should be able to include all custom code via Composer.
 
 When reviewing your site, take stock of exactly what versions of modules you depend on. One way to do this is to use a command like the following from within a contributed modules folder (e.g. `/modules`, etc):
 
-```bash{outputLines:2-4}
-find . -maxdepth 2 -name '*.info.yml' \
-  -exec basename '{}' '.info.yml' ';' \
-  -exec grep -e '^version' '{}' ';' \
-  -exec echo ';'
+```bash{promptUser:user}
+terminus drush $site.dev -- pm:projectinfo --fields=name,version --format=table
```

Comment:
Note that `pm:projectinfo` was removed in Drush 9, so the users will need to revert back to Drush 8 if they are using a newer Drush. Usually folks stay on Drush 8 until they convert to Composer, so this might not be a common problem.

Reply:
offered `pm:list` and linked to drush docs